packages/link-bins/test/index.ts

```diff
@@ -15,7 +15,15 @@ import tempy from 'tempy'
 
 jest.mock('@pnpm/logger', () => {
   const debug = jest.fn()
-  return () => ({ debug })
+  const globalWarn = jest.fn((...args) => {
+    console.warn(...args)
```

Comment:
don't print to console.

you can check that globalWarn was called with the right text in the test

Reply:
oh right, this was here when I'm trying to figure out how to mock 😅 

I've removed it now, I don't think it's that useful to assert warning message too (since message may change and even the slightest message change will fail the test)